src/components/select/select.template.tsx

```diff
@@ -91,6 +91,8 @@ const SelectValue: FunctionalComponent<
             aria-haspopup="listbox"
             aria-expanded="false"
             aria-labelledby="s-label s-selected-text"
+            aria-required={props.required}
+            aria-disabled={props.disabled}
```

Comment:
This is according to MDC guidelines: https://github.com/material-components/material-components-web/tree/69fd619d0cf28dc6588650282bc825858b104a74/packages/mdc-select